node/test/lib/relay_network.js

```diff
@@ -199,9 +204,11 @@ RelayNetwork.prototype.setCluster = function setCluster(cluster) {
                         cn: serviceName
                     },
                     hasNoParent: true
-                },
-                peers: self.topology[serviceName] // TODO this really ought to be linked directly to the root peers
+                }
             });
+            // The subchannel's peers are linked to the peer list as managed by
+            // incoming connections from the relays.
+            subChannel.peers = channel.peers;
```

Comment:
We should do this in Hyperbahn. The premise of using a ring is that its shape will adapt. The static list of hosts is only a seed list for bootstrapping. Once a service has found a member of the ring to talk to, the egress affinity peers on the ring will call back and may or may not be members of the seed list. The service should be sending all requests back to the ring on whatever peers the ring creates.